modules/local/create_pedfile.nf

```diff
@@ -0,0 +1,38 @@
+process MAKE_PED {
+    tag "make_ped"
+
+    conda (params.enable_conda ? "conda-forge::python=3.9.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.9--1' :
+        'quay.io/biocontainers/python:3.9--1' }"
+
+    input:
+    path samplesheet
```

Comment:
Rather than reading the samplessheet again, can't you get the information you need from `CH_INPUT.samples`? https://github.com/nf-core/raredisease/blob/e9dff85eddb7d8a8d15df15afcfd567b69577bbb/subworkflows/local/check_input.nf#L56 
It would be nice to only have one place where you parse the samplesheet and checks that everything is correct. That way we also don't need to change this code if we for some reason update the sample sheet with a new column or similar. 
Maybe one could even use nextflow scripting to write the file you need? Haven't really checked the feasibility to do that though.